subsys/bluetooth/host/monitor.c

```diff
@@ -174,8 +174,13 @@ static void encode_drops(struct bt_monitor_hdr *hdr, uint8_t type,
 
 static uint32_t monitor_ts_get(void)
 {
-	return (k_cycle_get_32() /
-		(sys_clock_hw_cycles_per_sec() / MONITOR_TS_FREQ));
+	int64_t cycle;
```

Comment:
```suggestion
	uint64_t cycle;
```

`k_cycle_get_64` returns a `uint64_t`